extensions.json

```diff
@@ -40,5 +40,19 @@
     "sha256sum": "1193589eb2779a1224328defca4e2ca378ef786474be1842ac43b674b9535d82",
     "foxe": "https://github.com/rscova/foxglove-sideways-teleop-extension/releases/download/v0.0.1/sideways.sideways-teleop-extension-0.0.1.foxe",
     "keywords": ["sideways","teleop"]
+  },
+    {
+    "id": "danielclapp.ros2-parameter-extension",
+    "name": "ROS2 Parameters",
+    "description": "Interact with parameters in ROS2",
+    "publisher": "Daniel Clapp",
+    "homepage": "https://github.com/danclapp4/ros2-parameter-extension.git",
+    "readme": "https://raw.githubusercontent.com/danclapp4/ros2-parameter-extension/main/README.md",
+    "changelog": "https://raw.githubusercontent.com/danclapp4/ros2-parameter-extension/main/CHANGELOG.md",
+    "license": "UNLICENSED",
```

Comment:
Please choose an [OSI-approved license](https://opensource.org/licenses) for extensions published in the public marketplace. MIT license is a good default if you're not sure which to pick.

Reply:
Ok I added MIT to the license field